pandas/tests/tools/test_to_datetime.py

```diff
@@ -933,6 +933,20 @@ def test_to_datetime_dtarr(self, tz):
         result = to_datetime(arr)
         assert result is arr
 
+    @td.skip_if_no("pyarrow")
+    @pytest.mark.parametrize("utc", [True, False])
+    @pytest.mark.parametrize("tz", [None, "US/Central"])
+    def test_to_datetime_arrow(self, tz, utc):
+        import pyarrow as pa
```

Comment:
```suggestion
        pa = pytest.importorskip("pyarrow")
```

Then you can remove `@td.skip_if_no("pyarrow")`

Reply:
Thanks, updated.